test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -18,11 +17,11 @@ public class LoggingTests : IISFunctionalTestBase
         [ConditionalTheory]
         [InlineData("CheckErrLogFile")]
         [InlineData("CheckLogFile")]
-        public async Task CheckStdoutLogging(string path)
+        public async Task CheckStdoutLoggingToFile(string path)
         {
             var deploymentParameters = Helpers.GetBaseDeploymentParameters();
             deploymentParameters.PublishApplicationBeforeDeployment = true;
-            deploymentParameters.PreservePublishedApplicationForDebugging = true; // workaround for keeping
+            deploymentParameters.PreservePublishedApplicationForDebugging = true; // workaround for keeping published output for log files
```

Comment:
Would this waste disk space infinitely? Set stdout location to a temporary file outside publish location.